x-pack/plugins/features/server/feature_schema.ts

```diff
@@ -53,10 +55,11 @@ const kibanaPrivilegeSchema = Joi.object({
   ui: Joi.array().items(Joi.string().regex(uiCapabilitiesRegex)).required(),
 });
 
-const kibanaSubFeaturePrivilegeSchema = Joi.object({
+const kibanaIndependentSubFeaturePrivilegeSchema = Joi.object({
   id: Joi.string().regex(subFeaturePrivilegePartRegex).required(),
   name: Joi.string().required(),
   includeIn: Joi.string().allow('all', 'read', 'none').required(),
+  minimumLicense: Joi.string().valid(...validLicenses),
```

Comment:
Observation: We don't attempt to prevent consumers from registering sub-feature privileges with a lower minimum license level than their parent privileges.

I don't think that we necessarily need to add that level of validation, but I wanted to point it out for posterity / in case it was an oversight.

Reply:
Another observation: The entire sub-feature privileges feature (ha!) is only available in Gold+. So, really, 'basic', 'standard', and 'gold' shouldn't be valid options... WDYT?

Reply:
> We don't attempt to prevent consumers from registering sub-feature privileges with a lower minimum license level than their parent privileges.
> 
> I don't think that we necessarily need to add that level of validation, but I wanted to point it out for posterity / in case it was an oversight.

I agree, I feel like validation for this is overkill, but thanks for pointing this out!

> The entire sub-feature privileges feature (ha!) is only available in Gold+. So, really, 'basic', 'standard', and 'gold' shouldn't be valid options... WDYT?

Makes a lot of sense, I'll make this change!